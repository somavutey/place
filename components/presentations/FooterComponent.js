import React from "react";
import Link from "next/link";
import { makeStyles } from "@mui/styles";
import { Grid, Typography } from "@mui/material";
const useStyles = makeStyles({
  footerSub: {
    display: "block",
    fontSize: "15px",
    lineHeight: "2em",
    "&:hover": {
      textDecoration: "underline",
    },
  },
});
const FooterListComponent = ({ children }) => {
  const classes = useStyles();
  return (
    <Link href="#">
      <a className={classes.footerSub}>{children}</a>
    </Link>
  );
};

export default FooterListComponent;
