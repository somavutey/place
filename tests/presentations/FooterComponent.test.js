import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import FooterComponent from "./../../components/presentations/FooterComponent";

it("FooterComponent", () => {
  const wrapper = shallow(<FooterComponent />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
