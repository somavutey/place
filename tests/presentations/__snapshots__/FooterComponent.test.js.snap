// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterComponent 1`] = `
<Link
  href="#"
>
  <a
    className="makeStyles-footerSub-1"
  />
</Link>
`;
